nucypher/blockchain/eth/deployers.py

```diff
@@ -202,8 +328,7 @@ class DispatcherDeployer(ContractDeployer):
     contract_name = DISPATCHER_CONTRACT_NAME
     deployment_steps = ('contract_deployment', )
     _upgradeable = False
-
-    DISPATCHER_SECRET_LENGTH = 32
+    _secret_length = 32
```

Comment:
Any particular reason for this change from `DISPATCHER_SECRET_LENGTH`? - I'm just curious